Pseudocode/Queue/Kth Largest Element in Stream/SourceCode.tex

```diff
@@ -0,0 +1,51 @@
+% Set the Page Layout
+\documentclass[12pt]{article}
+\usepackage[inner = 2.0cm, outer = 2.0cm, top = 2.0cm, bottom = 2.0cm]{geometry}
+
+% Package to write pseudo-codes
+\usepackage{algorithm}
+
+% Remove the 'end' at the end of the algorithm
+\usepackage[noend]{algpseudocode}
+
+% Define Left Justified Comments
+\algnewcommand{\LeftComment}[1]{\Statex \(\triangleright\) #1}
+
+% Remove the Numbering of the Algorithm
+\usepackage{caption}
+\DeclareCaptionLabelFormat{algnonumber}{Algorithm}
+\captionsetup[algorithm]{labelformat = algnonumber}
+
+\begin{document}
+
+\begin{algorithm}
+
+  \caption{Kth Largest Element in a Stream}
+  \begin{algorithmic}[1]
+    \Statex
+    \Function{Init\_Queue}{$array$, $k$}
+        \LeftComment $queue$ buffers k elements from the stream.
```

Comment:
Try not to clutter the functions. You can include clarifications at the top. Basically, the code block and the function signature should not be too far.